BREAKING.md

```diff
@@ -36,6 +37,10 @@ Previously, the fluidErrorCode value (a pascaleCased term) was often used as the
 Now all error messages can be expected to be easily-read sentences,
 sometimes followed by a colon and an inner error message when applicable.
 
+
+### Disallowed access to deleted sub directory
+Users will now be disallowed to do operations on a deleted directory. Users can subscribe to `dispose` event to know if a sub directory is deleted.
```

Comment:
might be worth to clarify what "disallow" actually means in practice

Reply:
fixed.